src/measure_theory/measure_space.lean

```diff
@@ -2436,6 +2436,18 @@ begin
   exact ⟨f', measurable.mono hf'_meas hm le_rfl, hff'.symm⟩,
 end
 
+lemma ae_measurable_restrict_of_measurable_subtype [h : nonempty β] {s : set α}
```

Comment:
you shouldn't need `nonempty β`, as all functions into the empty space are measurable (cf `measurable_of_not_nonempty`, although this is not exactly what you need).